src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -581,6 +584,17 @@ public synchronized boolean reserve(List<LockableResource> resources,
 		return true;
 	}
 
+	public synchronized boolean steal(List<LockableResource> resources,
```

Comment:
Potential nit: would `stealResources` (as acting on many) be better? There is precedent around for both naming patterns however...

Reply:
I'm guilty of same in #144 however. Guess having same names throughout Java and Jelly and request args was more straightforward ;)

Reply:
I stuck to the layout presented to me assuming I started writing functionality for the button (while copying what the reserve button does); as such that was how all the functions ended up being named.